client/dogs/FormPage.js

```diff
@@ -405,6 +127,55 @@ export default class IndexPage extends React.Component {
 
   handleSubmit = (ev: SyntheticInputEvent) => {
     ev.preventDefault();
+    let results = null;
+    const {
+      address,
+      ageMonths,
+      ageYears,
+      apt,
+      breed,
+      color,
+      comments,
+      dogName,
+      email,
+      firstName,
+      lastName,
+      neighborhood,
+      phone,
+      //rabiesCertFile,
+      rabiesExpire,
+      rabiesIssued,
+      sex,
+      //spayingCertFile,
+      zip,
+    } = this.state;
+
+    results = this.props.dogLicensesDao.mutate({
+      address,
+      ageMonths,
+      ageYears,
+      apt,
+      breed,
+      color,
+      comments,
+      dogName,
+      email,
+      firstName,
+      lastName,
+      neighborhood,
+      phone,
+      //rabiesCertFile,
+      rabiesExpire,
+      rabiesIssued,
+      sex,
+      //spayingCertFile,
+      zip,
+    });
+    //console.log(results);
+
+    return {
```

Comment:
don't need to return out of `handleSubmit`, no one is paying attention to the value.